test/e2e/gcloud_iam_roles.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2021 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Google Cloud IAM roles for E2E tests.
+#
+# Set of permissions required by the TriggerMesh end-to-end test suite to run
+# tests that manipulate Google Cloud resources.
+
+---
+
+# Permissions required by the test binary.
+name: projects/cebuk-01/roles/TriggerMeshE2E
+title: TriggerMesh E2E test suite
+description: Grants permissions required by the TriggerMesh E2E test suite.
+includedPermissions:
+- pubsub.topics.create
+- pubsub.topics.delete
+- pubsub.topics.publish
+- source.repos.create
+- source.repos.delete
+- storage.buckets.create
+- storage.buckets.delete
+- storage.objects.create
+- storage.objects.delete
```

Comment:
Permissions required by the test suite to run are documented in this file. It is the counterpart of `aws_iam_policy.jsonc` for Google Cloud.

I encourage everyone to keep it up-to-date.